webui/react/src/utils/job.ts

```diff
@@ -31,23 +33,30 @@ export const jobTypeToCommandType = (
   }
 };
 
-export const taskFromJob = (job: Job): AnyTask => {
-  const baseTask = { id: job.entityId, name: job.name };
-  let rv: AnyTask;
-  if (job.type === JobType.EXPERIMENT) {
-    rv = { ...baseTask, archived: false, state: RunState.Active } as ExperimentTask;
-  } else {
-    rv = {
-      ...baseTask,
-      state: CommandState.Running,
-      type: jobTypeToCommandType(job.type),
-    } as CommandTask;
-  }
-  return rv;
-};
-
 export const jobStateToLabel: {[key in JobState]: string} = {
   [JobState.SCHEDULED]: 'Scheduled',
   [JobState.SCHEDULEDBACKFILLED]: 'ScheduledBackfilled',
   [JobState.QUEUED]: 'Queued',
 };
+
+export const orderedSchedulers = new Set(
+  [ Api.V1SchedulerType.PRIORITY, Api.V1SchedulerType.KUBERNETES ],
+);
+
+export const moveJobToPositionUpdate = (jobId: string, position: number): Api.V1QueueControl => {
+  if (position < 1 || position % 1 !== 0) {
+    const err: DaError = {
+      message: `Invalid queue position: ${position}`,
+      type: ErrorType.Input,
+    };
+    throw err;
```

Comment:
**suggestion:** don't worry about this as part of this PR. `DaError` is probably best as a class that wraps around `Error` so that it can inherit everything from `Error`, and it can be declared and used in the same manner as `Error`.

e.g. `throw new DaError({ message: ..., type: ... })`

Reply:
yeah, that'd work nicely. 👍🏼